src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,58 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import duke.exception.DukeException;
+
+/**
+ * Deals with loading and saving data to the hard disk
+ */
+public class Storage {
+    private File directory;
+    private String filepath;
+
+    /**
+     * Constructs a storage to be managed at the specified file location.
+     *
+     * @param directoryPath The path of the directory containing the file
+     * @param filename The filename of the file to save the data to
```

Comment:
Perhaps you should add a full stop after every parameter description (see Java Coding Standard).